src/ProjectTemplates/Web.ItemTemplates/content/MvcController/.template.config/localize/templatestrings.en.json

```diff
@@ -3,7 +3,9 @@
   "name": "MVC Controller",
   "description": "MVC Controller with or without read/write actions",
   "symbols/namespace/description": "namespace for the generated code",
+  "_symbols/namespace/description.comment": "Should this start w/ 'Namespace'?",
```

Comment:
Yes, let's capitalize the first letters in all those sentences where you've raised the question.

Reply:
done